messaging/methods/messages/createMessage.md

```diff
@@ -482,7 +509,15 @@ var a = b;
 {% sample lang="python" %}
 
 ```python
-# Python Example
+body = MessageRequest()
+body.application_id = "93de2206-9669-4e07-948d-329f4b722ee2"
+body.to = ["+12345678902", "+12345678903"]
+body.mfrom = "+12345678901"
+body.text = "Hey, check this out!"
+try:
+    messaging_client.create_message("12345", body)
```

Comment:
Can you add example of working with the response object?

Reply:
I put the example in 1 of them. I think I just forgot to copy-paste it to the other ones. Updated 👍